src/lib/data/buyables/buyables.ts

```diff
@@ -717,6 +717,16 @@ const questBuyables: Buyable[] = [
 		name: 'Ring of shadows',
 		gpCost: 75_000,
 		requiredQuests: [QuestID.DesertTreasureII]
+	},
+	{
+		name: 'Crystal saw',
+		gpCost: 75_000,
+		qpRequired: 7,
+		skillsNeeded: {
+			agility: 25,
+			construction: 5,
+			magic: 46
+		}
 	}
```

Comment:
Isnt it made using a seed? and uses charges?

Reply:
Yes it's made using a seed, I could make the seed the buyable and make a saw a createable but that seems very long winded.

Also charges are used yes but it's free to recharge, so again, would just seem a long route about it.

Reply:
As a thought, maybe it could be taken out the build command as its not viable, and just kept in the pohcommand and stash unit command?